src/corelibs/U2Core/src/tasks/ModifySequenceObjectTask.cpp

```diff
@@ -200,6 +200,9 @@ void ModifySequenceContentTask::cloneSequenceAndAnnotations() {
         }
     }
     docs.append(newDoc);
+    if (AppContext::getProject() != nullptr) {
```

Comment:
Could you please explain how this line changes incorrect annotation coordinates?

Reply:
Check `FixAnnotationsUtils::fixAnnotations()` method, it can't find target annotation table if document not in the project. So as document will be added to project after editing anyway, i decided to add it right after creation.
We can do it in other way - remove condition `if (AppContext::getProject() != nullptr)` from `FixAnnotationsUtils::fixAnnotations()` and always search 'docs' for target annotation table.
As for me there is no much difference.

Reply:
I see, 
the good solution would be remove any dependency on project state from FixAnnotationUtils: pass all loaded annotation table objects in project as a parameter to it. 

This way it will not depend if the document is added to the project or not.